### ISSUE TITLE
All received data (timezone_offset)

### ISSUE THREAD

**Issue by Optega** [csparpa/pyowm#427]
https://github.com/csparpa/pyowm/blob/51d669a3f2de339d053bdc2460bfed7e5b33a083/pyowm/weatherapi25/one_call.py#L105

It would be cool to add all the data that is returned, especially "timezone_offset", it will help to set the time correctly

**Comment by csparpa** [comment]
I agree on adding the timezone offset - though, I'm not actively developing on PyOMW
Would you like to work out a Pull Request yourself please @Optega ?
